src/schema/nmdc.yaml

```diff
@@ -14,8 +14,12 @@ license: https://creativecommons.org/publicdomain/zero/1.0/
 version: 0.0.0
 
 imports:
+  - linkml:types
+  - attribute_values_classes
   - annotation # also brings core and portal_*
   - workflow_execution_activity
+  - basic_classes
+  - basic_slots
```

Comment:
can't these be imported indirectly through other imports? i'd liek to keep it succinct.

Reply:
Which sub-files would you like the basic_classes and basic_slots to come in via?